### PR TITLE
[ENG-5762] Get GV set up in osf docker configs

### PR DESCRIPTION
**Pull request by opaduchak**
## Purpose
The purpose of this pr is to integrate gravyvalet into osf's docker-compose.yml for more straightforward development experience

## Changes

- Added `gv` and `gv-requirements` services to `docker-compose.yml`
- Added `.docker-compose.gv.env` file for gravyvalet environment definition

## QA Notes

It'd be great for other folks to try it out before merge, to mitigate "works on my machine" type of risks 

## Ticket
[ENG-5762](https://openscience.atlassian.net/browse/ENG-5762)

[ENG-5762]: https://openscience.atlassian.net/browse/ENG-5762?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by aaxelb — CHANGES_REQUESTED** [review]
good start! could you also update `README-docker-compose.md` to match? might be as simple as adding `gv` next to `wb` and `gv_requirements` next to `wb_requirements`, but i forget what all is in there

File: .docker-compose.gv.env [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ALLOWED_HOSTS="localhost,192.168.168.167"
+CORS_ALLOWED_ORIGINS="http://localhost:5000,http://192.168.168.167:5000"
+OSFDB_HOST=192.168.168.167
+POSTGRES_HOST="$OSFDB_HOST"
+OSF_BASE_URL="http://192.168.168.167:5000"
+OSF_API_BASE_URL="http://192.168.168.167:8000"
+POSTGRES_USER="postgres"
+POSTGRES_DB="gravyvalet"
```
[the remark this answers is not in the document]
Comment (opaduchak) [reply .docker-compose.gv.env:8]: added postgres SQL init script to create `gravyvalet` db if not exists

File: README-docker-compose.md [hunk 2]
```diff
@@ -222,7 +226,9 @@
     - _NOTE: CTRL-c will exit_
 - Run migrations:
   - After creating migrations, resetting your database, or starting on a fresh install you will need to run migrations to make the needed changes to database. This command looks at the migrations on disk and compares them to the list of migrations in the `django_migrations` database table and runs any migrations that have not been run.
-    - `docker-compose run --rm web python3 manage.py migrate`
+    - `docker-compose run --rm web python3 manage.py migrate` To run `osf` migrations
+    - `docker-compose run --rm gv python manage.py migrate` To run `gravyvalet(gv)` migrations
+  - After creating migrations, resetting your database, or starting on a fresh install you will need to run migrations to make the needed changes to database. This command looks at the migrations on disk and compares them to the list of migrations in the `django_migrations` database table and runs any migrations that have not been run.
```
Comment (aaxelb) [inline README-docker-compose.md:231]: nit: this line is duplicated (tho i agree it makes more sense after the `migrate` commands

**Review by aaxelb — APPROVED** [review]
only a docs nit, otherwise looks good (TIL psql's `\gexec` -- little indirect but works for me)

**Review by aaxelb — APPROVED** [review]
🦑

**Pull request merged**